src/model/WinPlayer.cs

```diff
@@ -33,21 +33,29 @@ public void Stop()
         }
 
         /// <inheritdoc/>
-        public void Play(string path)
+        public void OpenFile(string path)
         {
-            if (this.outputDevice.PlaybackState == PlaybackState.Stopped)
-            {
-                this.audioFileReader = new AudioFileReader(path);
-                this.outputDevice.Init(this.audioFileReader);
```

Comment:
With this solution, the audio file loads silently and then waits for the user to start playback using the Play/Pause button. 

I had originally been modeling the behavior after [VLC's](https://www.videolan.org/vlc/index.html) open file menu item behavior, and so for consistency I'm going to add this.outputDevice.Play(); to the line below so that the audio plays once a user has loaded a file.